dev/releases/update_website.py

```diff
@@ -80,21 +80,21 @@ def download_asset_by_name(asset_name: str, writedir: str) -> None:
         utils.download_with_sha256(url, asset_name)
 
 
-def extract_tarball(tarball: str) -> None:
```

Comment:
Why is `tarball` renamed to `tball`? (No objection, just wondering what the rationale is)

Reply:
Because `tarball` is a variable defined in an outer scope, according to pylint.